interface/themes/style_light.css

```diff
@@ -1492,3 +1492,8 @@ table.dataTable#therapy_groups_list tbody td{
   margin: 20px 20px 2px 20px;
 
 }
+
+.sidebar {
+  background: #f5f5f5;
+  border-right: 1px solid #eee;
+}
```

Comment:
No, other styles will still work. This is just making the sidebar look better on the Light theme